.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Type of Issue:
+Some example types are:
+* bug in science or answers
+* bug in software
+* science enhancement (changes answers and requires science development)
+* software enhancement
+* FATES change
+* Discussion
+
+
+## Summary of Issue:
```

Comment:
Often the summary of the issue appears in the subject / title line, so doesn't need to be called out separately in the body.